#! /bin/sh -e
## 10_a_capella.dpatch by  <kyle@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix typo in A Capella genre

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad /home/debian/mad/libid3tag-0.15.1b/libid3tag-0.15.1b/genre.dat libid3tag-0.15.1b/genre.dat
--- /home/debian/mad/libid3tag-0.15.1b/libid3tag-0.15.1b/genre.dat	2004-02-16 21:34:39.000000000 -0500
+++ libid3tag-0.15.1b/genre.dat	2004-09-06 09:50:33.000000000 -0400
@@ -277,8 +277,8 @@
   { 'P', 'u', 'n', 'k', ' ', 'R', 'o', 'c', 'k', 0 };
 static id3_ucs4_t const genre_DRUM_SOLO[] =
   { 'D', 'r', 'u', 'm', ' ', 'S', 'o', 'l', 'o', 0 };
-static id3_ucs4_t const genre_A_CAPPELLA[] =
-  { 'A', ' ', 'C', 'a', 'p', 'p', 'e', 'l', 'l', 'a', 0 };
+static id3_ucs4_t const genre_A_CAPELLA[] =
+  { 'A', ' ', 'C', 'a', 'p', 'e', 'l', 'l', 'a', 0 };
 static id3_ucs4_t const genre_EURO_HOUSE[] =
   { 'E', 'u', 'r', 'o', '-', 'H', 'o', 'u', 's', 'e', 0 };
 static id3_ucs4_t const genre_DANCE_HALL[] =
@@ -452,7 +452,7 @@
   genre_DUET,
   genre_PUNK_ROCK,
   genre_DRUM_SOLO,
-  genre_A_CAPPELLA,
+  genre_A_CAPELLA,
   genre_EURO_HOUSE,
   genre_DANCE_HALL,
   genre_GOA,
diff -urNad /home/debian/mad/libid3tag-0.15.1b/libid3tag-0.15.1b/genre.dat.in libid3tag-0.15.1b/genre.dat.in
--- /home/debian/mad/libid3tag-0.15.1b/libid3tag-0.15.1b/genre.dat.in	2004-01-23 04:41:32.000000000 -0500
+++ libid3tag-0.15.1b/genre.dat.in	2004-09-06 09:50:33.000000000 -0400
@@ -153,7 +153,7 @@
 Duet
 Punk Rock
 Drum Solo
-A Cappella
+A Capella
 Euro-House
 Dance Hall
 Goa
